fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -418,6 +418,8 @@ def build_settings(key: nil, optional: true)
 
         command = build_xcodebuild_showbuildsettings_command
 
+        command = "#{command} 2>&1" # xcodebuild produces errors on stderr #21672
+
```

Comment:
I wonder if it wouldn't make more sense to bake this `2>&1` within the implementation of `build_xcodebuild_showbuildsettings_command` instead? 🤔